From 0bdd9922d6829bb8da8168d2b2fad59eca550080 Mon Sep 17 00:00:00 2001
From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Thu, 28 Sep 2017 09:58:59 -0300
Subject: [PATCH] doc: fix build file

It was missing a comma, and breaking the build with
--enable-gtk-doc
---
 doc/reference/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/reference/meson.build b/doc/reference/meson.build
index 1e59f14..46b1f76 100644
--- a/doc/reference/meson.build
+++ b/doc/reference/meson.build
@@ -58,7 +58,7 @@ gnome.gtkdoc(
           mkdb_args: [ '--sgml-mode', '--output-format=xml' ],
         html_assets: html_images,
             install: true,
-        install_dir: doc_path
+        install_dir: doc_path,
   gobject_typesfile: meson.project_name() + '.types',
           scan_args: [
     '--rebuild-types',
